### ISSUE TITLE
There are different left/right padding amounts for the new DataToolbar component and the individual list rows.

### ISSUE THREAD

**Issue by jlmitch5** [ansible/awx#5628]
BACKLOG'D and marked WONTFIX by patternfly

There are different left/right padding amounts for the new DataToolbar component and the individual list rows. This is a known issue, and patternfly was going to update the CSS so that the padding is consistent. [issue](https://github.com/patternfly/patternfly-next/issues/2503) currently open to address this in patternfly-core (css)

**Comment by jlmitch5** [comment]
just updated and checked on how these things are going:

> There are different left/right padding amounts for the new DataToolbar component and the individual list rows. This is a known issue, and patternfly will be updating CSS so that the padding is consistent. issue currently open to address this in patternfly-core (css)

this seems like it is in prioritized backlog

> Overflow of keys and tag values in the search tags section needs some work. Currently the values ellipses, but there's no tooltip to show the full value. fixed issue for adding ellipses in patternfly-next, and for adding tooltip to label, tooltip for value still not finished

last part of this is the tooltip for the value, which is assigned, but no fix merged yet.  probably will be fixed soon.

> There are various small issues being addressed with the visual design of the DataToolbar components in this PR. We should be able to take advantage of these fixes without any code changes once they are included in a PF release and we update.

this should be fixed.  not sure if it's in the vesion we have installed for ui_next or not.

> When the data toolbar components are moved to stable from experimental, we'll need to update the import statements.

looks like they are still marked beta

**Comment by jlmitch5** [comment]
all fixed except for the padding one

**Comment by mabashian** [comment]
Im not sure that this is still blocked by PF

**Comment by unlikelyzero** [comment]
closing to track in https://github.com/ansible/awx/issues/7825
